samples/go/README-ptbr.md

```diff
@@ -0,0 +1,54 @@
+# Lead Integration - Go
+
+[English version](README.md)
+
+Um simples projeto usando Go para demonstrar como receber leads do zap+.
+
+Por favor, confira nossa [documentação](https://developers.grupozap.com/) para mais informações.
+
+## Recursos Utilizados
+
+- [Go 1.16](https://golang.org/)
+- [Gorilla Mux 1.8](https://github.com/gorilla/mux)
+- [Joho GoDotEnv 1.3](https://github.com/joho/godotenv)
```

Comment:
I think [ligeiro](https://github.com/olxbr/ligeiro) can help you with env vars. Just a suggestion :)